### DISCUSSION TITLE
Stop deploying on push

### DISCUSSION THREAD

**Discussion by devkennyy** [vercel/vercel#8178] [Help] [answered]
Is it possible to disable this and only deploy manually?

**Comment by amyegan** [comment, marked as answer]
Hi @devkennyy. You can disable all GitHub deployments for your Project by setting [github.enable](https://vercel.com/docs/project-configuration#git-configuration/github-enabled) to false in `vercel.json`. If you still want automated deployments in some cases but not others, you can do that with [ignored build step](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel) configuration.
